src/finch/compiled.py

```diff
@@ -27,8 +27,23 @@ def lazy(tensor: Tensor):
         return Tensor(jl.Finch.LazyTensor(tensor._obj))
     return tensor
 
-
-def compute(tensor: Tensor, *, verbose: bool = False):
+def set_optimizer(opt="default"):
+    if opt == "default":
+        jl.Finch.set_scheduler_b(jl.Finch.default_scheduler())
+    elif opt == "galley":
+        jl.Finch.set_scheduler_b(jl.Finch.galley_scheduler())
+    return
+
+def clear_optimizer_cache():
```

Comment:
This should not be necessary with the latest update to Finch in https://github.com/finch-tensor/Finch.jl/pull/649

Reply:
Do you mean the separate functions or the clearing of the cache?

I just checked and the cache is still being hit without the clearing.

Reply:
Have you pulled the latest finch? I just want to double check. It was my intention that Finch 1.0.0 would clear the cache if you use a different tag, but not if you use the same tag. the default does not clear the cache,

Reply:
if you want to clear it every time, you may want to give it a different tag every time

Reply:
I am not sure if I will be able to clear the cache efficiently in future versions

Reply:
Ah, I see. I need to add tag parameters to this then. That makes sense. I'll update it with that.